src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -814,17 +814,29 @@ if bAPIError then
 end
 
 -- Set default settings
+settings.set( "bios.use_multishell", true )
 settings.set( "shell.allow_startup", true )
 settings.set( "shell.allow_disk_startup", (commands == nil) )
 settings.set( "shell.autocomplete", true )
+settings.set( "shell.promptColor", "yellow" )
+settings.set( "shell.textColor", "white" )
+settings.set( "shell.backgroundColor", "black" )
 settings.set( "edit.autocomplete", true ) 
 settings.set( "edit.default_extension", "lua" )
+settings.set( "edit.backgroundColor", "black" )
+settings.set( "edit.textColor", "white" )
+settings.set( "edit.highlightColor", "yellow" )
+settings.set( "edit.keywordColor", "yellow" )
+settings.set( "edit.commentColor", "green" )
+settings.set( "edit.stringColor", "red" )
 settings.set( "paint.default_extension", "nfp" )
 settings.set( "lua.autocomplete", true )
+settings.set( "lua.promptColor", "yellow" )
+settings.set( "lua.textColor", "white" )
```

Comment:
I really think we should move this to 'underscore_case'. I believe settings should be consistent with each other.